okhttp/src/commonMain/kotlin/okhttp3/internal/http/HttpMethod.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2014 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.http
+
+expect object HttpMethod {
```

Comment:
Can’t this be implemented in commonMain? Expect/actual is so annoying and we should try to make things common wherever we have the chance.

Reply:
Relevant https://github.com/square/okhttp/pull/5218 - but apparently JvmStatic works in common.